core/src/main/java/org/elasticsearch/indices/cache/query/terms/TermsLookup.java

```diff
@@ -56,21 +63,118 @@ public String getId() {
         return id;
     }
 
+    public String getPath() {
```

Comment:
last super small comment, can we remove the get prefix from these setters from now, for consistency reasons?